v1.5/getting-started/bare-metal/bare-metal.md

```diff
@@ -141,74 +141,57 @@ Therefore, we recommend creating a failsafe Calico security policy that
 is tailored to your environment. The example commands below show one
 example of how you might do that; the commands:
 
--   Create a new policy tier called "failsafe" with `order` 0. When
-    creating other tiers, you should give them a higher `order` value so
-    the failsafe rules get applied first.
--   Add a single policy to that tier, which
-
-    > -   applies to all known endpoints
-    > -   allows inbound ssh access from a defined "management" subnet
-    > -   allows outbound connectivity to etcd on a particular IP; if
-    >     you have multiple etcd servers you should duplicate the rule
-    >     for each destination
-    > -   allows inbound ICMP
-    > -   allows outbound UDP on port 67, for DHCP
-    > -   uses a "next-tier" action to pass any non-matching packets to
-    >     the next tier of policy.
+- Add a single policy, which
+  - applies to all known endpoints
+  - allows inbound ssh access from a defined "management" subnet
+  - allows outbound connectivity to etcd on a particular IP; if
+    you have multiple etcd servers you should duplicate the rule
+    for each destination
+  - allows inbound ICMP
+  - allows outbound UDP on port 67, for DHCP.
 
 <!-- -->
 
-    etcdctl set /calico/v1/policy/tier/failsafe/metadata '{"order": 0}'
-    etcdctl set /calico/v1/policy/tier/failsafe/policy/failsafe \
+    etcdctl set /calico/v1/policy/tier/default/policy/failsafe \
```

Comment:
This should be done with `calicoctl`, given it's a key feature of the new version :)